app/assets/stylesheets/molecules/_show-more.scss

```diff
@@ -0,0 +1,65 @@
+// Original implementation (with specs):
+// https://github.com/codeforamerica/gcf-backend/commit/410a93ee58b69c88919a2a59d73a9320fc952004
+
+.show-more {
+  text-align: left;
+  margin-bottom: 3em;
+  position: relative;
+
+  &:after {
+    content: '';
+    display: block;
+    position: absolute;
+    bottom: 0em;
+    left: 0;
+    right: 0;
+    background-image: linear-gradient(-180deg, rgba(255,255,255,0.00) 0%, #FFFFFF 75%);
+    height: 7em;
+    border-bottom: 2px solid #EAEAE9; // $color-light-grey
+  }
+
+  &.is-open {
+    border-bottom: 0;
+
+    &:after {
+      display: none;
+    }
+    .show-more__content {
+      max-height: none;
+    }
+    .show-more__button {
+      display: none;
+    }
+  }
+}
+
+.show-more--long {
+  .show-more__content {
+    max-height: 30rem;
+  }
+}
+
+.show-more__button {
+  font-size: 1.6rem; // $font-size-small
```

Comment:
Are these commented variables just what they were called in GCF?

Reply:
yeah, also our variables file is in the styleguide and isn't by default available to stuff in our application, so we were getting a compilation error when trying to use the variables by name

I imagine we will want to pull this into the styleguide, so wanted to leave the variables easily findable so left them as comments.